recipes/PySimpleGUIWeb/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pysimpleguiweb" %}
+{% set version = "0.35.0" %}
+{% set styledName = "PySimpleGUIWeb" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz
+  sha256: "2cb85ec488d3046ba8300a8b6f58a2475e4cff6bc208b8303cec9052df823a70"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.4
+    - remi
+
+  run:
+    - python >=3.4
+
+test:
+  imports:
+    - PySimpleGUIWeb
+  requires:
+    - remi
+    - python
+
+about:
+  home: "https://github.com/PySimpleGUI/PySimpleGUI"
+  license: "LGPL-3.0"
+  license_family: "LGPL"
+  license_file: "license.txt"
+  summary: "GUI SDK Launched in 2018 Actively developed and supported. Super-simple to create custom GUI's.  Python 2.7 & 3 Support. 100 Demo programs & Cookbook for rapid start. Extensive documentation. Examples using Machine Learning(GUI, OpenCV Integration,  Chatterbot), Rainmeter Style Floating Desktop Widgets, Matplotlib + Pyplot integration, add GUI to command line scripts, PDF & Image Viewer. Great for beginners as well as advanced GUI programmers"
```

Comment:
Replace the `summary` here with `description`, and add `summary` with only the first sentence.